lib/adapter/udp.js

```diff
@@ -111,8 +111,7 @@ adapter.send = function (message, callback) {
       }
     ],
     function (err, result) {
-      client.close();
-      return callback(err, result);
+      client.close(callback(err, result));
```

Comment:
This code is not really valid. 1. callback will be executed instantly; 2. the close function accepts a callback.

Most probably what you want to see is something like this: `client.close(function () { callback(err, result); });` or `client.close(() => callback(err, result));`

see the ticket description for more details.

Reply:
Good catch. Will resubmit.